dietpi/boot

```diff
@@ -123,49 +115,43 @@
 	Run_Init
 
 	#----------------------------------------------------------------
-	#Pre-Installed image, 1st run
+	# Pre-installed image, 1st run
 	if (( $G_DIETPI_INSTALL_STAGE == 2 )); then
 
 		# - Set swap
 		/DietPi/dietpi/func/dietpi-set_dphys-swapfile $(grep -m1 '^[[:blank:]]*AUTO_SETUP_SWAPFILE_SIZE=' /DietPi/dietpi.txt | sed 's/^[^=]*=//') "$(grep -m1 '^[[:blank:]]*AUTO_SETUP_SWAPFILE_LOCATION=' /DietPi/dietpi.txt | sed 's/.*=//')"
 
 		# - Run survey
-		rm /DietPi/dietpi/.dietpi-survey &> /dev/null
-		/DietPi/dietpi/dietpi-survey 1 &> /dev/null &
+		[[ -f /DietPi/dietpi/.dietpi-survey ]] && rm /DietPi/dietpi/.dietpi-survey
+		/DietPi/dietpi/dietpi-survey 1 &
 
 		# - Continue with normal boot
 		export G_DIETPI_INSTALL_STAGE=1
 		echo $G_DIETPI_INSTALL_STAGE > /DietPi/dietpi/.install_stage
 
 	fi
 
-	#Normal Boot
+	# Normal boot
 	if (( $G_DIETPI_INSTALL_STAGE == 1 )); then
 
-		/DietPi/dietpi/func/run_ntpd 1 &> /dev/null &
+		# - Run time sync
+		/DietPi/dietpi/func/run_ntpd 1 &
```

Comment:
Needs to be redirected to null:
```
root@DietPi:~# /DietPi/dietpi/func/run_ntpd 1 &
[1] 13013
[  OK  ] Root access verified.
[  OK  ] DietPi-Run_ntpd | systemctl restart systemd-timesyncd
[ INFO ] DietPi-Run_ntpd | NTPD: Waiting for completion of systemd-timesyncd (1/60)
[ INFO ] DietPi-Run_ntpd | NTPD: Waiting for completion of systemd-timesyncd (2/60)
[ INFO ] DietPi-Run_ntpd | NTPD: Waiting for completion of systemd-timesyncd (3/60)
[ INFO ] DietPi-Run_ntpd | NTPD: Waiting for completion of systemd-timesyncd (4/60)
[ INFO ] DietPi-Run_ntpd | NTPD: Waiting for completion of systemd-timesyncd (5/60)
[ INFO ] DietPi-Run_ntpd | NTPD: Waiting for completion of systemd-timesyncd (6/60)
[ INFO ] DietPi-Run_ntpd | NTPD: Waiting for completion of systemd-timesyncd (7/60)
[ INFO ] DietPi-Run_ntpd | NTPD: Waiting for completion of systemd-timesyncd (8/60)
[  OK  ] DietPi-Run_ntpd | NTPD: systemd-timesyncd synced
[  OK  ] NTPD: time sync | Completed
```

Reply:
@Fourdee 
I thought being a bid more verbose here would be nice, for debugging and to see what is going on during boot. Also for testing reasons.
But I see the issue with too much output (seconds tick here) messing with other output, when jobs running in background. Also processing animation could be an issue. The cannot occur concurrently, but would look ugly. Will re-add redirection, makes sense for all background calls.